### PR TITLE
Add About Menu to TopBar for swagger-editor@next

### LINKED ISSUE #3646: SwaggerEditor@next: add About Section TopBar
Add `About` section for the new editor and tie off some additional SEO loose ends.

### PR DESCRIPTION
**Pull request by frankkilcommins**
Updated Home anchor link + added new About menu to TopBar which links to various documentation and product related material for Swagger-Editor@Next

fixes #3646
closes #3646

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "analyze": "source-map-explorer 'build/static/js/main.js'",
     "test": "react-scripts test",
     "cy:dev": "start-server-and-test cy:dev:server http://localhost:3003 cy:dev:open",
-    "cy:dev:open": "CYPRESS_BASE_URL=http://localhost:3003 cypress open --e2e",
+    "cy:dev:open": "cross-env CYPRESS_BASE_URL=http://localhost:3003 cypress open --e2e",
```
Comment (char0n) [inline package.json:44]: Thanks for catching this.

File: src/plugins/top-bar/components/AboutMenu/AboutMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+
+const AboutMenu = ({ getComponent }) => {
+  const DropdownMenu = getComponent('DropdownMenu');
+  const Link = getComponent('Link');
+
+  return (
+    <>
+      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
```
Comment (char0n) [inline src/plugins/top-bar/components/AboutMenu/AboutMenu.jsx:9]: No prop spreading happens here, we can drop the comment.
```suggestion
```

File: src/plugins/top-bar/components/AboutMenu/AboutMenu.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <DropdownMenu label="About">
+        <li>
```
Comment (char0n) [inline src/plugins/top-bar/components/AboutMenu/AboutMenu.jsx:11]: DropdownMenuItem should have the following form:

```jsx
<div>
  <li className="dropdown-item">
    <Link href="https://swagger.io/tools/swagger-editor/" target="_blank">
      About Swagger Editor
     </Link>
  </li>
</div>
```

Can you align the code to reflect this?

**Review by char0n — CHANGES_REQUESTED** [review]
@frankkilcommins can you also provide screenshot how the editor looks like after incorporating my feedback with opened `About` menu?

**Comment by frankkilcommins** [comment]
> @frankkilcommins can you also provide screenshot how the editor looks like after incorporating my feedback with opened `About` menu?

@char0n 
image

**Review by char0n — APPROVED** [review]
LOOKS good

**Pull request merged**